system/service.py

```diff
@@ -403,10 +403,31 @@ def get_service_tools(self):
                 self.svc_initscript = initscript
 
         def check_systemd():
-            return os.path.exists("/run/systemd/system/") or os.path.exists("/dev/.run/systemd/") or os.path.exists("/dev/.systemd/")
+
+            # tools must be installed
+            if location.get('systemctl',False):
+
+                # this should show if systemd is the boot init system
+                # these mirror systemd's own sd_boot test http://www.freedesktop.org/software/systemd/man/sd_booted.html
+                for canary in ["/run/systemd/system/", "/dev/.run/systemd/", "/dev/.systemd/"]:
```

Comment:
No, checking for `systemd/user` doesn't tell us anything about `systemd` running as init service. I suggest always checking for `systemd/system`.